localstack-core/localstack/services/lambda_/event_source_mapping/pollers/sqs_poller.py

```diff
@@ -216,3 +216,14 @@ def get_queue_url(queue_arn: str) -> str:
     account_id = parsed_arn["account"]
     name = parsed_arn["resource"]
     return f"{host}/{account_id}/{name}"
+
+
+def message_attributes_to_lower(message_attrs):
+    """Convert message attribute details (first characters) to lower case (e.g., stringValue, dataType)."""
```

Comment:
nit: for brevity, we could actually just say "Converts keys fom `PascalCase` to `camelCase`" 🙂

Reply:
another inlining from here: https://github.com/localstack/localstack/pull/11733/files#diff-7e0d439c4f8162098a496d085987f824965da98bdcdd38e983354c03c2c6fa21L193
We prefer to keep the changes minimal and therefore I refrained from changing/improving anything not strictly necessary.